src/graph/workflow.ts

```diff
@@ -460,18 +460,23 @@ export class Workflow {
     private redrawEdges() {
 
         const edgeEls             = this.model.connections.filter(el => el.isVisible);
-        const highlightedEdgesMap = new Map<Edge, boolean>();
+        const highlightedEdgesMap = new Map<string, boolean>();
```

Comment:
If using strings as keys, we don't need a Map object anymore, we can just store it in a plain `{}`.
Using `JSON.stringify` seems a bit like an overkill here, can you just form an id like
```
el.attributes["data-source-connection"].value + el.attributes["data-destination-connection"].value
```

and 

```
c.source.id + c.destination.id
```